test/unit/markdown-service-test.js

```diff
@@ -280,14 +280,63 @@ describe('Markdown Service', () => {
 
     it('should convert single digit numbered lists to add \\\\ between the digit and the digit\'s "."', () => {
```

Comment:
These descriptions show 4 slashes vs. 2 to ensure the user sees the correct number of slashes in the description when run `npm test`

With 2 slashes:
 ✓ should convert single digit numbered lists to add \\ between the digit and the digit's "."

With 4 slashes:
✓ should convert single digit numbered lists to add \\\\ between the digit and the digit's "."